tools/webpack.config.ts

```diff
@@ -1,6 +1,7 @@
 import { resolve } from 'path';
 import * as BundleTracker from 'webpack-bundle-tracker';
 import * as webpack from 'webpack';
+import * as webpackDevServer from 'webpack-dev-server';
```

Comment:
This is what causes the failure in travis ci because `webpack-dev-server` is a marked as a dev dependency hence not installed during the production install workflow. You can't reproduce this failure locally since `tools/provision` installs `webpack-dev-server`.